core/src/main/java/org/lflang/validation/AttributeSpec.java

```diff
@@ -213,10 +214,12 @@ enum AttrParamType {
     ATTRIBUTE_SPECS_BY_NAME.put(
         "side",
         new AttributeSpec(List.of(new AttrParamSpec(VALUE_ATTR, AttrParamType.STRING, false))));
-    // @enclave(each=boolean)
+    // @enclave(workers=inteach=boolean)
```

Comment:
Should this be:
```suggestion
    // @enclave(workers=int, each=boolean)
```
What is the meaning of the each parameter?